### DISCUSSION TITLE
Sort account names by name or rank

### DISCUSSION THREAD

**Discussion by Chaziz** [rejetto/hfs#747] [Ideas]
Basically in the accounts management page, to be able to sort accounts alphabetically or by rank (admin).
As you can see in my picture it is not sorted alphabetically.

![Captura de pantalla 2024-09-21 214847](https://github.com/user-attachments/assets/d3d4e795-8094-41a0-8ef6-7512769b8181)

**Comment by rejetto** [comment]
oh, right.
it will be fixed in next version.
you can see it in this preview version 
https://github.com/user-attachments/files/17089249/hfs-linux-x64-0.54.0-alpha1.4.zip
https://github.com/user-attachments/files/17089250/hfs-windows-x64-0.54.0-alpha1.4.zip

to install, go to Admin > Home > right-click on check-for-update

**Reply by Chaziz** [reply]
thank you rejetto!
